lib/multidb/log_subscriber.rb

```diff
@@ -11,7 +11,7 @@ def sql(event)
     def debug(msg = nil)
       name = Multidb.balancer.current_connection_name
       if name
-        db = color("[DB: #{name}]", ActiveSupport::LogSubscriber::GREEN, true)
+        db = color("[DB: #{name}]", ActiveSupport::LogSubscriber::GREEN, bold: true)
```

Comment:
We need to ensure this works on more than Rails 7.1+.

can you put a check to ensure this form is being used on rails 7.1+ only?